examples/pubsub/tutorial_pubsub_mqtt_publish.c

```diff
@@ -471,7 +475,20 @@ int main(int argc, char **argv) {
     }
     addDataSetWriter(server, topic);
 
-    UA_Server_run(server, &running);
+    UA_Server_run_startup(server);
+    size_t iteration = 0;
+    while(running) {
+        iteration++;
+        UA_Server_run_iterate(server, true);
+        if(iteration % 100 == 0) {
+            /* Trigger publishing of DataSetMetaData. In a real application we would have made 
+             * changes to the properties of the corresponding node at this point, 
+             * which results in an update of the MinorVersion of the DataSetMetaData's ConfigurationVersion
+             * and thus, a new DataSetMetaData message is sent. */ 
+            UA_Server_updateDataSetField(server, fieldNodeId, UA_TRUE, UA_FALSE);
+        }
+    }
+    UA_Server_run_shutdown(server);
```

Comment:
This code-path is very complex and execution dependet. Please use a cyclic callback to e.g. publish the metada every 5 seconds.

Reply:
Done